arm-software/embedded/llvmlibc-samples/ldscripts/llvmlibc-raspi3b.ld

```diff
@@ -0,0 +1,18 @@
+/*
```

Comment:
Sorry to be a pain, could you add the Copyright and license information here too?